### PR TITLE
Improving creation of ballots (fixes #3103)

### LINKED ISSUE #3103: Creating a ballot needs a lot of time
- create an election with e.g. 25 candidates and project it
- change the phase from "Searching for candidates" to "Voting"
- start a new new/first ballot

result: it takes on my PC 10 seconds until this action is finished. During this time the browser is frozen. On my laptop it takes up to 20 seconds.

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
This speeds up the creation a lot. (26 candidates. Before 12 s, after 2 s)

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
Confirmed. Successfully tested with 22 candidates.

**Comment by FinnStutzenstein** [comment]
More things changed:
- Just one autoupdate for mass creation of speakers when creating a ballot.
- Just one autoupdate for mass deletion ('delete all speakers from list').

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein travis failed:
`ERROR: test_remove_someone_else (tests.unit.agenda.test_views.ItemViewSetManageSpeaker)`

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Any Idea how to avoid the inform_changed_data in the test?
I tried with
```
@patch('openslides.utils.autoupdate.inform_changed_data')
...
mock_inform_changed_data.return_value = None
def side_effect(*args, **kwargs):
    print('Hello')
mock_inform_changed_data.side_effect = side_effect
```
But this does not work.

File: openslides/poll/models.py [hunk 1]
```diff
@@ -131,7 +132,8 @@ def set_options(self, options_data=[]):
         for option_data in options_data:
             option = self.get_option_class()(**option_data)
             option.poll = self
-            option.save()
+            option.save(skip_autoupdate=True)
+        inform_changed_data(self.get_assignment())
```
Comment (normanjaeckel) [inline openslides/poll/models.py]: Sorry but you can't reference `assignment` in this abstract poll module.

Comment (FinnStutzenstein) [inline openslides/poll/models.py]: Done.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Ping.

**Pull request merged**